pkg/systemd/util.go

```diff
@@ -0,0 +1,58 @@
+//go:build linux
```

Comment:
The file name should be _linux.go

Reply:
Heh, I was considering  that, but also knew there were some other PR's being reviewed for "no_systemd" build-tags, so thought I could reduce some churn by using build-tags inside the file, but happy to rename if it's a blocker for you ☺️